### PR TITLE
refactor: update import logic

### PR DESCRIPTION
**Pull request by hetao92**

### CODE REVIEW

**Review by mizy — APPROVED** [review]
LGTM

File: app/pages/Import/FileUpload/FileList.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                title={intl.get('common.ask')}
+                okText={intl.get('common.ok')}
+                cancelText={intl.get('common.cancel')}
+              >
+                <Button className="warningBtn" type="link">
+                  <Icon type="icon-studio-btn-delete" />
+                </Button>
+              </Popconfirm>
+            </div>
+          );
```
Comment (huaxiabuluo) [inline app/pages/Import/FileUpload/FileList.tsx:55]: file.content may be empty/falsy ?
```ts
if (!file.content) {
  return null;
}

return (
  <div>...</div>
);
```

File: app/pages/Import/FileUpload/index.tsx [hunk 2]
```diff
@@ -1,115 +1,64 @@
...
+    }).catch(_err => {
+      setLoading(false);
     });
-    setLoading(false);
   };
+
+  const handleDelete = async (index: number) => {
+    const file = fileList[index].name;
+    await deleteFile(file)
+  }
```
Comment (huaxiabuluo) [inline app/pages/Import/FileUpload/index.tsx:44]: unnecessary `async`?
```ts
const handleDelete = (index: number) => {
  const fileName = fileList[index].name;
  deleteFile(fileName);
}
```

File: app/stores/index.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ import dataImport from './import';
 import schema from './schema';
 import graphInstances from './graphInstances';
 
-const rootStore = { global, files, console, dataImport, schema, graphInstances };
+const rootStore = { global, files, console, dataImport, schema, graphInstances } as any;
```
Comment (huaxiabuluo) [inline app/stores/index.ts:9]: why `any` ?

**Pull request merged**